<def f='include/llvm-6.0/llvm/Support/raw_ostream.h' l='442' type='void llvm::raw_fd_ostream::clear_error()'/>
<doc f='include/llvm-6.0/llvm/Support/raw_ostream.h' l='433'>/// Set the flag read by has_error() to false. If the error flag is set at the
  /// time when this raw_ostream&apos;s destructor is called, report_fatal_error is
  /// called to report the error. Use clear_error() after handling the error to
  /// avoid this behavior.
  ///
  ///   &quot;Errors should never pass silently.
  ///    Unless explicitly silenced.&quot;
  ///      - from The Zen of Python, by Tim Peters
  ///</doc>
